3_visualize/src/diagnostics.R

```diff
@@ -0,0 +1,34 @@
+#' Export model diagnostics
+#' @export model_diagnostics_text txt, file contains mean RSME for three model types as well as a breakdown of training temperature profiles for the different models 
+
+diagnostics <- function(){
+  
+  # Check for existence of out subfolder
+  dir.exists("3_visualize/out")
+  
+  # Read in eval data 
+  eval_data <- readRDS("2_process/out/eval_data")
```

Comment:
I'm calling it out here, but this comment applies to all the functions you've written - `in` and `out` files are great candidates for arguments. it's nice to have them as easy access so you can quickly change around file structure